clients/admin-ui/src/features/privacy-requests/privacy-requests.slice.ts

```diff
@@ -353,6 +353,32 @@ export const privacyRequestApi = createApi({
       }),
       invalidatesTags: ["Notification"],
     }),
+    getStorageDetails: build.query<any, any>({
```

Comment:
Oh no does this whole file use `any` for its query params/returns? I'd expect the models to be generated by [openapi](https://github.com/ethyca/fides/blob/5a485387d8af247ec6479e4115088cbbb8394d77/clients/admin-ui/src/types/api/README.md#L0-L1).